cli/webfinger.ts

```diff
@@ -74,6 +95,21 @@ class InvalidHandleError extends Error {
   }
 }
 
+/**
+ * Custom error class for invalid max redirection values.
+ * @param {number} maxRedirection The invalid max redirection value.
+ * @extends {Error}
+ */
+class InvalidMaxRedirectionError extends Error {
```

Comment:
Instead of defining `InvalidMaxRedirectionError`, you could utilize [Cliffy's `ValidationError`](https://cliffy.io/docs@v1.0.0-rc.7/command/error-handling#custom-validation-errors).